cmd/get/social/social.go

```diff
@@ -31,32 +58,120 @@ func NewsocialCmd() *cobra.Command {
 	}
+	fl := cmd.Flags()
+	fl.BoolP("all", "a", false, "option to get all providers")
+	fl.BoolP("active", "c", false, "option to get active providers")
 
 	return cmd
 }
 
 func get() error {
+	if temp == "all" {
+		resultResp1, err := GetAllFields()
+		if err != nil {
+			return err
+		}
+		var num int
+		for i := 0; i < len(resultResp1.Data); i++ {
+			fmt.Print(fmt.Sprint(i+1) + ".")
+			fmt.Println(resultResp1.Data[i].Display)
+		}
+		// Taking input from user
+		fmt.Print("Please select a number from 1 to " + fmt.Sprint(len(resultResp1.Data)) + " :")
+		fmt.Scanln(&num)
+		for 1 > num || num > len(resultResp1.Data) {
+			fmt.Print("Please select a number from 1 to " + fmt.Sprint(len(resultResp1.Data)) + " :")
+
+			fmt.Scanln(&num)
+		}
+		fmt.Println("Display: " + resultResp1.Data[num-1].Display)
+		fmt.Println("Enabled: ", resultResp1.Data[num-1].Configured)
+		fmt.Println("IsMandatory: ", resultResp1.Data[num-1].Mdfile)
+		fmt.Println("Parent: ", resultResp1.Data[num-1].Name)
+		fmt.Println("ParentDataSource: ", resultResp1.Data[num-1].Options)
+		fmt.Println("Permission: ", resultResp1.Data[num-1].Order)
+		fmt.Println("Name: ", resultResp1.Data[num-1].Scopes)
+		fmt.Println("Options: ", resultResp1.Data[num-1].Selected)
+
+	}
+	if temp == "active" {
+		resultResp, err := GetActiveFields()
+		if err != nil {
+			return err
+		}
+		var num int
+		for i := 0; i < len(resultResp.Data); i++ {
+			fmt.Print(fmt.Sprint(i+1) + ".")
+			fmt.Println(resultResp.Data[i].Provider)
+		}
+		// Taking input from user
+		fmt.Print("Please select a number from 1 to " + fmt.Sprint(len(resultResp.Data)) + " :")
+		fmt.Scanln(&num)
+		for 1 > num || num > len(resultResp.Data) {
+			fmt.Print("Please select a number from 1 to " + fmt.Sprint(len(resultResp.Data)) + " :")
+
+			fmt.Scanln(&num)
+		}
+		fmt.Println("Display: " + resultResp.Data[num-1].HtmlFileName)
+		fmt.Println("Enabled: ", resultResp.Data[num-1].Provider)
+		fmt.Println("IsMandatory: ", resultResp.Data[num-1].ProviderId)
+		fmt.Println("Parent: ", resultResp.Data[num-1].ProviderKey)
+		fmt.Println("ParentDataSource: ", resultResp.Data[num-1].ProviderSecret)
+		fmt.Println("Permission: ", resultResp.Data[num-1].Scope)
+		fmt.Println("Name: ", resultResp.Data[num-1].Status)
+	}
+
+	return nil
+}
+
+func GetActiveFields() (*socialProviderList, error) {
 	conf := config.GetInstance()
+	Url = conf.AdminConsoleAPIDomain + "/platform-configuration/social-providers/options?"
 
-	url = conf.AdminConsoleAPIDomain + "/platform-configuration/social-providers/options?"
+	var R1 socialProviderList
+	resp, err := request.Rest(http.MethodGet, Url, nil, "")
 
-	var resultResp socialProviderList
-	resp, err := request.Rest(http.MethodGet, url, nil, "")
 	if err != nil {
-		return err
+		return nil, err
 	}
-	err = json.Unmarshal([]byte(resp), &resultResp)
+
+	err = json.Unmarshal(resp, &R1)
+	if err != nil {
+		return nil, err
+	}
+	return &R1, nil
+}
+
+func GetAllFields() (*allSocialProviderList, error) {
```

Comment:
We don't need this method, define a static list of available social provider and show them as per the plan
```
For Free there are 3 
For developer, there are 5
```
Please check in the dashboard once and add a static struct